filament/backend/include/private/backend/CommandStream.h

```diff
@@ -38,6 +38,7 @@
 #include <tuple>
 #include <thread>
 #include <utility>
+#include <cstddef>
```

Comment:
Please keep includes information alphabetical order

Reply:
please use `<stddef.h>`. We don't use the c++ version of the C headers when possible.